### ISSUE TITLE
Unable to run project from VS: "No valid targets for run"

### ISSUE THREAD

**Issue by developer82** [appcelerator/vscode-appcelerator-titanium#142]
### Description
Running my project via VS was working fine. I debugged the project, and stopped the debugger but the terminal that was running the application was stuck on running without any option to close it and run my application normally, so I had closed VS entirely and started it again.
Now when I try to run my project I get the following error message:
```
2019-09-29T06:47:20.654Z | ERROR  | No valid targets for run. Are you in a project folder?
```

My project is located in my F drive at `f:\development\myproject` but when running I see that VS generates the command with a wrong project folder:
```
appc "run" "--platform" "android" "--log-level" "info" "--project-dir" "C:\development\myproject" "--target" "emulator" "--device-id" "Nexus_5_API_26"  
```
It uses C drive instead of F drive for some reason. I tried to restart VS but no luck. Running from command line inside the project folder and without `--project-dir` parameter runs the project successfully.

**Comment by ewanharris** [comment]
This is probably caused by d08138abfcca62caff35bbecf3048aef87888499, I think maybe we need to do a proper `path.parse` on the project directory to get the root rather than just resolving `/`

**Comment by developer82** [comment]
I wonder why it started suddenly and was working fine before the case I have described above

**Comment by ewanharris** [comment]
We only shipped that change Friday. I guess VS Code updated the extension in the background and then when you restarted the new version was loaded with this bug

**Comment by ewanharris** [comment]
@developer82, could I get you to try the fix in #143 if you have time please? If you open up `~/.vscode/extensions/axway.vscode-titanium-0.6.0/out/utils.js`  replace the `normalizeDriveLetter` function with the below and then restart your editors it should pick up the change

```js
function normalizeDriveLetter(filePath) {
    if (process.platform !== 'win32') {
        return filePath;
    }
    const { root } = path.parse(filePath);
    return `${root.substr(0, 1).toUpperCase()}${filePath.slice(1)}`;
}
```

**Comment by ewanharris** [comment]
@developer82, we just released 0.6.1 which should fix this issue
